src/context/UserDataContext/properties/lastVisit.ts

```diff
@@ -31,46 +36,62 @@ export default class LastVisitProperty extends UserDataPropertyAPI {
       this.getLocalStorageKey('consecutiveVisits'),
       this.consecutiveVisits
     );
+    this.saveLocalStorageValue(
+      this.getLocalStorageKey('totalNoVisits'),
+      this.totalNoVisits
+    );
   };
 
   eraseFromLocalStorage = () => {
     window.localStorage.removeItem(this.getLocalStorageKey('lastVisitDate'));
     window.localStorage.removeItem(
       this.getLocalStorageKey('consecutiveVisits')
     );
+    window.localStorage.removeItem(this.getLocalStorageKey('totalNoVisits'));
   };
 
   exportValue = (): any => {
     return {
       lastVisitDate: this.lastVisitDate,
       consecutiveVisits: this.consecutiveVisits,
+      totalNoVisits: this.totalNoVisits,
     };
   };
 
   importValueFromObject = (data: object) => {
     this.lastVisitDate = data['lastVisitDate'] || new Date().getTime();
     this.consecutiveVisits = data['consecutiveVisits'] || 1;
+    this.totalNoVisits = data['totalNoVisits'] || 1;
   };
 
   getAPI = () => {
     return {
       lastVisitDate: this.lastVisitDate,
       consecutiveVisits: this.consecutiveVisits,
+      totalNoVisits: this.totalNoVisits,
       setLastVisitDate: (today: number) => {
         let timeSinceLastVisit = today - this.lastVisitDate;
         let oneDay = 1000 * 60 * 60 * 20,
           twoDays = 1000 * 60 * 60 * 24 * 2;
 
         let newLastVisit = null,
-          newConsecutiveVisits = null;
-
+          newConsecutiveVisits = null,
+          newTotalNoVisits = null;
+        newTotalNoVisits = this.totalNoVisits + 1;
```

Comment:
we don't need a `newTotalNoVisits` variable -- the total # of visits will always change